modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/java/com/liferay/frontend/editor/ckeditor/web/internal/editor/configuration/BaseCKEditorConfigContributor.java

```diff
@@ -80,11 +76,7 @@ public void populateConfigJSONObject(
 					PortalUtil.getStaticResourceURL(
 						themeDisplay.getRequest(),
 						"/o/frontend-editor-ckeditor-web/ckeditor/skins" +
-							"/moono-lisa/editor.css")),
-				HtmlUtil.escape(
-					PortalUtil.getStaticResourceURL(
-						themeDisplay.getRequest(),
-						"/o/frontend-css-web/main.css")))
+							"/moono-lexicon/editor.css")))
```

Comment:
Interesting that we need this here... 🤔

Reply:
Do we need this here?

Reply:
Let me check again

Reply:
Yep, we need it:

![image](https://user-images.githubusercontent.com/5803434/86601964-4adfa780-bfa2-11ea-915e-860b8853cd1c.png)

Reply:
So how did it use to work before? Were we also adding the `moono.css` file manually? It doesn't feel 100% right. I'd expect CKEditor's `ContextMenu` plugin to be already fetching the skin... but I could be wrong here...

Reply:
Oh, wait, before applying the new styles we were not passing any `contextmenu_contentsCss` param to CKEditor, we added it to apply the styles from portal, probably we're overwriting it. Checking what happen if we remove the param.